### PR TITLE
ENT-1477: Adding client_subclients endpoint

### PR DESCRIPTION
**Pull request by Vitor-Carvalho**
This PR will just add a new endpoint to return all the subclients of a client (GET /client/subclients).

rubocop:
151 files inspected, no offenses detected

TEST:
pry(main)> bw.client_subclients
=> {"resultsTotal"=>-1,
 "resultsPage"=>-1,
 "resultsPageSize"=>-1,
 "results"=>
  [{"id"=>45,
    "name"=>"CL1",
    "parentId"=>44,
    "startDate"=>"2015-03-19T00:00:00.000+0000",
    "expiryDate"=>nil,
    "website"=>"",
    "address1"=>"",
    "address2"=>"",
    "address3"=>"",
    "country"=>"",
    "postcode"=>"",
    "contactName"=>"",
    "contactEmail"=>"vitor+c@brandwatch.com",
    "contactTitle"=>"",
    "contactTelephone"=>"",
    "contactMobile"=>"",
    "mentionArchiveSize"=>240,
    "tags"=>{"ratelimit.limit"=>"30", "ratelimit.period"=>"10", "resellerTermsAndConditionsAccepted"=>"true", "resellerTermsAndConditionsAcceptedBy"=>"1372", "resellerTermsAndConditionsTimestamp"=>"Thu Mar 19 12:15:59 UTC 2015"},
    "maximumSubscribedBrands"=>0,
    "pricingModel"=>"QB",
    "priceStructureId"=>1159,
    "priceStructureLineId"=>1169,
    "maxUsers"=>0,
    "theme"=>"brandwatch",
    "owner"=>"vitor@brandwatch.com",
    "isLegacy"=>false,
    "mentionBasedPricingMatrixLevel"=>0,
    "userInactivityTimeoutMinutes"=>1440},
   {"id"=>62,
    "name"=>"CL6",
    "parentId"=>44,
    "startDate"=>"2015-04-30T00:00:00.000+0000",
    "expiryDate"=>nil,

### CODE REVIEW

**Comment by Vitor-Carvalho** [comment]
@jonathanchrisp , @benSlaughter : could you please have a look at this?

File: lib/bwapi/client/client.rb [hunk 1]
```diff
@@ -11,6 +11,12 @@ def client
         get 'client'
       end
 
+      # Get the current users client subclients
+      #
```
Comment (jonathanchrisp) [inline lib/bwapi/client/client.rb]: can we add a return comment here?

Comment (Vitor-Carvalho) [inline lib/bwapi/client/client.rb]: Sure. I've updated the PR

**Comment by jonathanchrisp** [comment]
LGTM :+1:

**Pull request merged**